docs/installation.md

```diff
@@ -13,14 +13,17 @@ installed):
 	  - `clang`: required by RocksDB.
 	  - `protobuf`: required for protobuf serialization (gRPC).
 	  - `cmake`: required for building protobuf
+    - `git-lfs`: The Git extension for [Large File Support](https://git-lfs.github.com/) (required for EF tests submodule).
    6. Navigate to the working directory.
-   7. Run the test by using command `cargo test --all`. By running, it will pass all the required test cases.
+   7. If you haven't already, clone the repository with submodules: `git clone --recursive https://github.com/sigp/lighthouse`.
```

Comment:
What do you think about swapping 6 & 7, then changing _"Navigate to the working directory."_ to _"Navigate to the root of the repository."_?